zubbi/models.py

```diff
@@ -47,6 +47,51 @@
     "description": "description^1.2",
 }
 
+RESERVED_CHARACTERS = [
+    "+",
+    "-",
+    "=",
+    ">",
+    "<",
```

Comment:
I think since those chars can't be escaped but need to be removed we should also remove them from the list here.

Reply:
I did that first, but afterwards thought it might be a good idea to keep them in the list in case we need that list for something else. But most probably this isn't necessary. I'll remove them.